src/material/datepicker/datepicker-base.ts

```diff
@@ -232,10 +232,26 @@ export interface MatDatepickerControl<D> {
   stateChanges: Observable<void>;
 }
 
+/** A picker panel that can be associated with a {@link MatDatepickerControl}. */
+export interface MatDatepickerPanel<C extends MatDatepickerControl<D>, S,
+    D = ExtractDateTypeFromSelection<S>> {
+  closedStream: EventEmitter<void>;
+  color: ThemePalette;
+  datepickerInput: C;
+  disabled: boolean;
+  id: string;
+  opened: boolean;
+  openedStream: EventEmitter<void>;
+  stateChanges: Subject<void>;
+  open(): void;
+  registerInput(input: C): MatDateSelectionModel<S, D>;
```

Comment:
Since this is a public API, each of these members need a JsDoc description (these are auto generated into material.angular.io)

Reply:
Done